explorer/frontend/nixpkgs

```diff
@@ -0,0 +1 @@
+/nix/store/mz8k9ldyi6bz5wdq6xpmf3s3vqw3w5mn-61fbdb47a69f78998f55207e64122a0798047b5d.tar.gz-unpacked
```

Comment:
Are we sure this file is meant to be there? My hunch is that it was committed by mistake.